tests/behaviour/contracts/cross_contract_calls/invalid_output.sol

```diff
@@ -0,0 +1,28 @@
+
+pragma solidity ^0.8.10;
+
+//SPDX-License-Identifier: MIT
+
+
+
+interface I{
+    function invalid() external returns (uint8);
+}
+
+
+contract A{
+    /// warp-cairo
+    /// DECORATOR(external)
+    /// func invalid_bde9e40e() -> (number : felt) {
```

Comment:
we have a macro for current func, this will make it possible to not edit this test when we remove name mangling

Reply:
smart way to get around the solc type system!